### PR TITLE
Add command to get the last Sigma analysis of a file

### PR DESCRIPTION
**Pull request by danipv**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/VirusTotal/ReleaseNotes/2_4_0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### VirusTotal (API v3)
+- Added new Sigma command
+  - ***vt-file-sigma-analysis***
+Analists can now get the last Sigma analysis of a given file hash.
```
Comment (yaakovi) [inline Packs/VirusTotal/ReleaseNotes/2_4_0.md:6]:
```suggestion
    Analysts can now get the last Sigma analysis of a given file hash.
- Updated the Docker image to: *demisto/python3:3.10.4.28442*.
```

File: Packs/VirusTotal/Integrations/VirusTotalV3/README.md [hunk 2]
```diff
@@ -2014,3 +2014,113 @@ Retrieves resolutions of the given IP.
...
 >| u-20694f234fbac92b1dcc16f424aa1c85e9dd7af75b360745df6484dcae410853-1613980758 | harmless: 69<br/>malicious: 7<br/>suspicious: 0<br/>undetected: 7<br/>timeout: 0 | completed |
+
+### vt-file-sigma-analysis
+
+***
+Retrieves result of the last Sigma analysis.
+
+#### Base Command
+
+`vt-analysis-get`
```
Comment (yaakovi) [inline Packs/VirusTotal/Integrations/VirusTotalV3/README.md:2025]:
```suggestion
`vt-file-sigma-analysis`
```

File: Packs/VirusTotal/Integrations/VirusTotalV3/README.md [hunk 3]
```diff
@@ -2014,3 +2014,113 @@ Retrieves resolutions of the given IP.
...
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| file | File hash (md5, sha1, sha256). | Required |
+| only_stats | Print only Sigma analysis summary stats. | Optional |
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
```
Comment (yaakovi) [inline Packs/VirusTotal/Integrations/VirusTotalV3/README.md:2036]: add those outputs to the yml file as well

**Comment by danipv** [comment]
🤦🏼

**Review by yaakovi — APPROVED** [review]

**Pull request merged**